### DISCUSSION TITLE
A way to adjust texture offsets in the editor by dragging as well as direct number input

### DISCUSSION THREAD

**Discussion by ignomen** [FakeFishGames/Barotrauma#7252] [Ideas and suggestions]
I am making this post on Hexcode's request, and on behalf of Vectura.
The suggestion is:
To 'add a way [in the sub editor] to adjust texture offsets by dragging as well as direct number input'.

**Comment by MaaiG** [comment]
- Yes! It was very hard for me until I found out about the "Shift button pressed" while dragging.
From myself I want to offer the possibility of "merging objects" into groups - for moving. Combine them as layers in Photoshop. When tuning a submarine, it often takes a very long time to isolate the right objects and move them aside or back.
- There’s a big problem with rearranging the wires. If I have to move the wires aside, I have to move them one at a time. I would like to be able to isolate all the intermediate points of the wires in one place and move them simultaneously.
